superset/views/base.py

```diff
@@ -333,6 +342,14 @@ def serialize_bootstrap_data() -> str:
     return {"bootstrap_data": serialize_bootstrap_data}
 
 
+@superset_app.context_processor
+def get_html_template_script() -> Dict[str, Any]:
+    def process_template_script() -> str:
+        return render_template_string(superset_app.config["HTML_TEMPLATE_SCRIPT"])
```

Comment:
call render_template_string incase HTML_TEMPLATE_STRING contains template variables

Reply:
This is really cool !
But , should we have parallel support for HTML_STRING and HTML_TEMPLATE_STRING ?

For who does want to use template and can potentially reduce security risk by not enabling template

Reply:
unfortunately we're not going with this route (see discussion bellow), though your suggestion would definitely make sense for the current approach.